core/logic/src/main/java/org/apache/syncope/core/logic/init/ClassPathScanImplementationLookup.java

```diff
@@ -74,6 +75,8 @@
     private Map<Class<? extends AccountRuleConf>, Class<? extends AccountRule>> accountRuleClasses;
 
     private Map<Class<? extends PasswordRuleConf>, Class<? extends PasswordRule>> passwordRuleClasses;
+    
+    private List<String> basePackages = Arrays.asList("org.apache.syncope.fit.core", "org.apache.syncope.core");
```

Comment:
Prior to this change, one could create classes, in its own Maven-based Syncope project, under any package, now it is forced to do it under `org.apache.syncope.client.console` but we got more startup speed.
I guess it is an acceptable trade-off.

LGTM.